src/test/kotlin/calculator/CalculatorTest.kt

```diff
@@ -0,0 +1,83 @@
+package calculator
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+class CalculatorTest {
+    @Test
+    fun `정상 입력`() {
+        val calculator = Calculator()
+        assertThat(calculator.input("1111 * 2 + 1 - 3 / 2")).isEqualTo(1110)
+        assertThat(calculator.input("1111 * 2 * 10 * 3 * 2")).isEqualTo(133320)
+        assertThat(calculator.input("1111 + -2 + 10 + 3 * 2")).isEqualTo(2244)
+    }
+
+    @Test
+    fun `입력값이 null이거나 빈 공백 문자인 경우`() {
+        val calculator = Calculator()
+        Assertions.assertThrows(IllegalArgumentException::class.java) {
```

Comment:
```suggestion
        Assertions.assertThrows<IllegalArgumentException> {
```
좀더 간결한 형태로 타입을 넘길 수 있습니다. 이런 사용법을 알게 되는것을 넘어서 내부적으로 어떻게 가능한지도 알아 보실 수 있는 기회를 가지시면 좋겠습니다. 😀